src/test/java/com/iexec/sms/admin/AdminServiceTests.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms.admin;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class AdminServiceTests {
+
+    private final AdminService adminService = new AdminService();
+
+    @Test
+    void shouldReturnNotImplementedWhenCallingBackup() {
+        Assertions.assertEquals("createDatabaseBackupFile is not implemented", adminService.createDatabaseBackupFile());
+    }
+
+    @Test
+    void shouldReturnNotImplementedWhenCallingRestore() {
+        Assertions.assertEquals("restoreDatabaseFromBackupFile is not implemented", adminService.restoreDatabaseFromBackupFile("", ""));
+    }
```

Comment:
I would already add regions to limit merge conflicts and allow fine grained code folding from the start
```suggestion
    // region backup
    @Test
    void shouldReturnNotImplementedWhenCallingBackup() {
        Assertions.assertEquals("createDatabaseBackupFile is not implemented", adminService.createDatabaseBackupFile());
    }
    // endregion

    // region restore-backup
    @Test
    void shouldReturnNotImplementedWhenCallingRestore() {
        Assertions.assertEquals("restoreDatabaseFromBackupFile is not implemented", adminService.restoreDatabaseFromBackupFile("", ""));
    }
    // endregion
```